src/views/product-landing/components/overview-cta/index.tsx

```diff
@@ -9,20 +13,31 @@ function OverviewCta({
   image,
 }: OverviewCtaProps) {
   return (
-    <>
-      <pre className={s.placeholder}>
-        <code>
-          <h2 id={headingSlug} className={s.heading}>
-            {heading} [id={headingSlug}]
-          </h2>
-          {JSON.stringify(
-            { component: 'OverviewCta', heading, body, cta, image },
-            null,
-            2
-          )}
-        </code>
-      </pre>
-    </>
+    <div className={s.root}>
+      <div className={s.textPart}>
+        <h2 id={headingSlug} className={s.heading}>
+          {heading}
+        </h2>
+        <p className={s.body}>{body}</p>
+        <StandaloneLink
+          className={s.cta}
+          text={cta.text}
+          href={cta.url}
+          color="secondary"
+          icon={
+            isAbsoluteUrl(cta.url) ? (
+              <IconExternalLink16 />
+            ) : (
+              <IconArrowRight16 />
```

Comment:
Super tiny nit that I could see design bringing up, but we may want to add like `1px` padding on top of the icons to visually center them.

Reply:
Good eye, it does look a little high 🪴 I think we'd do this in `StandaloneLink`, right?

FWIW I _think_ we might better match the design spec without the extra `1px`:

<img width="1184" alt="2022-05-06-standalone-link-icon-align" src="https://user-images.githubusercontent.com/4624598/167210533-347dd563-459e-4577-afa9-d12523b54445.png">

Given how close it is, i've made a task to revisit this on the `QA` section of our tutorials board rather than try to tackle in this PR: 🎟️ [[standalone-link] maybe tweak icon position](https://app.asana.com/0/1202001387788447/1202239425737242/f)